fdbclient/include/fdbclient/NativeAPI.actor.h

```diff
@@ -560,8 +560,9 @@ ACTOR Future<std::vector<CheckpointMetaData>> getCheckpointMetaData(Database cx,
 // Checks with Data Distributor that it is safe to mark all servers in exclusions as failed
 ACTOR Future<bool> checkSafeExclusions(Database cx, std::vector<AddressExclusion> exclusions);
 
+// Round up to the nearest page size
 inline uint64_t getWriteOperationCost(uint64_t bytes) {
-	return bytes / std::max(1, CLIENT_KNOBS->WRITE_COST_BYTE_FACTOR) + 1;
+	return (bytes - 1) / CLIENT_KNOBS->WRITE_COST_BYTE_FACTOR + CLIENT_KNOBS->WRITE_COST_BYTE_FACTOR;
```

Comment:
I think this should be adding 1 to the whole quantity (or the byte factor to the numerator):

```suggestion
	return (bytes - 1) / CLIENT_KNOBS->WRITE_COST_BYTE_FACTOR + 1
```